src/components/InputPassword/InputPassword.css

```
@@ -0,0 +1,14 @@
+.Input-Password{
```

Please implement with Pascal Case
```suggestion
.InputPassword{
```

---

@Mubangizi this has been fixed.